docs/restarting.md

```diff
@@ -0,0 +1 @@
+# Resetting Chronicle
```

Comment:
Is this file intended?

Reply:
Thanks, no it's a duplicate and I've removed it